File: Angular/SimpleBankingSystem/src/app/admin-change-pin.ts
export class AdminChangePin {
    id?:any; 
     oldPassword?:any;
     newPassword?:any;

     constructor(  id?:any, oldPassword?:any, newPassword?:any){
        this.id=id;
        this.oldPassword=oldPassword;
        this.newPassword=newPassword;
     }
}

File: Spring Boot/bank/src/main/java/com/example/adminlogin/AdminChangePin.java
package com.example.adminlogin;

public class AdminChangePin {
	int id;
	 String  oldPassword, newPassword;

	public AdminChangePin() {
		super();
	}

	public AdminChangePin(String oldPassword, String newPassword, int id) {
		super();
		this.oldPassword = oldPassword;
		this.newPassword = newPassword;
		this.id = id;
	}

	public String getOldPassword() {
		return oldPassword;
	}

	public void setOldPassword(String oldPassword) {
		this.oldPassword = oldPassword;
	}

	public String getNewPassword() {
		return newPassword;
	}

	public void setNewPassword(String newPassword) {
		this.newPassword = newPassword;
	}

	public int getId() {
		return id;
	}

	public void setEmail(int id) {
		this.id = id;
	}
	 
	 
}
